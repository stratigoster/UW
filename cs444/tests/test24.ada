PACKAGE foo IS
  TYPE data;
  TYPE pdata IS ACCESS data; -- forward references work!
  TYPE data IS RECORD
    a,b,c,d,e : integer;
    f,g,h,a,i : boolean;
--        ^
-- conflicting declarations of variable `a'
  END RECORD;
END foo;
