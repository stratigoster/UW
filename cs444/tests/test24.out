***ERROR***:9:6:	conflicting declarations of variable  a

------
Compilation error
------
